src/components/Select/Select.js

```diff
@@ -47,6 +44,11 @@ const Select = ({
           className="bx--select__arrow"
           description={iconDescription}
         />
+        {!inline ? (
```

Comment:
Is moving the `<label>` here intentional?

Reply:
Yes, the html was updated in the vanilla repo, so this is just matching the changes to the v9 component there.